src/Elastic.Markdown/Myst/CodeBlocks/EnhancedCodeBlockParser.cs

```diff
@@ -84,11 +84,11 @@ public override bool Close(BlockProcessor processor, Block block)
 			"console" => "json",
 			"console-response" => "json",
 			"console-result" => "json",
-			"sh" => "bash",
-			"yml" => "yaml",
 			"terminal" => "bash",
 			_ => codeBlock.Language
 		};
+		if (!string.IsNullOrEmpty(codeBlock.Language) && !CodeBlock.Languages.Contains(codeBlock.Language))
+			codeBlock.EmitWarning($"Unknown language: {codeBlock.Language}");
```

Comment:
Can we actually list the supported languages or add a link to supported languages?